scanner/updater/manual/vulns.yaml

```diff
@@ -369,6 +369,24 @@ vulnerabilities:
 # Vuln: CVE-2025-24813/GHSA-83qj-6fr2-vhqg
 # Reason: The Tomcat JAR does not contain a pom.properties file, so cannot match the finding to OSV's data.
```

Comment:
can you also mention that very old versions may produce just `tomcat-embed-core` instead of the full `org.apache.tomcat-embed-core:tomcat-embed-core`?